docs/agents/agent-development.md

```diff
@@ -237,14 +237,33 @@ Global labels can be specified via the environment variable `ELASTIC_APM_GLOBAL_
 
 ### Transactions
 
+#### Transaction outcome
```

Comment:
Add link to table of contents

Reply:
@axw did you auto-generate the toc somehow?

Reply:
Yes, I used https://github.com/yzhang-gh/vscode-markdown. Probably not an ideal long term solution.